getting-started/compile-contract.md

```diff
@@ -20,9 +20,8 @@ to a pre-defined beneficiary. First, clone the repo and try to build the wasm bu
 # get the code
 git clone https://github.com/CosmWasm/cosmwasm-examples
 cd cosmwasm-examples
-git fetch --tags
-git checkout escrow-0.10.0
-cd escrow
+git checkout 8ff1161218c10028e7891bb2a0f096f094cb0c18
```

Comment:
I don't love using a commit hash here, but `escrow-0.10.0` is compiling to a newer version of the vm than the musselnet-4 testnet supports. I tried using `v0.9.0` but that also had issues.